src/source/quick-brick-login-multi-login-providers/QuickBrickMultiLoginProvidersVM.bs

```diff
@@ -0,0 +1,463 @@
+import "pkg:/source/UIBuilder/view/ScreenVM.bs"
+import "pkg:/source/roku_modules/maestro/core/Result.brs"
+import "pkg:/source/lib/APIUtils.bs"
+
+
+class QuickBrickMultiLoginProvidersVM extends zapp.uib.ScreenVM
+
+  public currentScreen = invalid
```

Comment:
Please order in to private and public. 

Put a header above the public fields. 

Put a header above the private fields.